WorkloadManagementSystem/Service/WMSUtilities.py

```diff
@@ -78,8 +79,8 @@ def getARCPilotOutput( proxy, pilotRef ):
   myce = pilotRef.split(":")[1].strip("/")
   gridEnv = getGridEnv()
   mySite = getCESiteMapping()['Value'][myce]
-  WorkDB = gConfig.getValue(os.path.join('Resources/Sites/LCG', mySite, 'CEs', myce, 'JobListFile'))
-  myWorkDB = os.path.join("/opt/dirac/runit/WorkloadManagement/SiteDirector-RAL2", WorkDB)
+  WorkDB = gConfig.getValue(cfgPath['Resources/Sites/LCG', mySite, 'CEs', myce, 'JobListFile'])
+  myWorkDB = os.path.join("/opt/dirac/runit/WorkloadManagement/SiteDirector-RAL", WorkDB)
```

Comment:
And yes - lines 103 - 107 are needed. They are the ones which actually return the pilot log to the screen.